### ISSUE TITLE
Build Lineage-13

### ISSUE THREAD

**Issue by IKGapirov** [ngoquang2708/android_local_manifests#1]
Good Day!
Thank you for your [**answer**](https://github.com/diepquynh1501/android_device_samsung_coreprimeve3g/issues/2)..
Very sorry I wrote here, because I do not know where to write to you
Again I`ll try to build LineageOS-13.0 for grandprimeve3g from this [**local_manifests**](https://github.com/diepquynh1501/local_manifests/blob/cm-13.0-grandprimeve3g/grandprimeve3g.xml)  but get [**error**](https://gist.github.com/IKGapirov/820019955f29e930ceed18c4401e1ed4) again((...
Can you please explain how to fix it? 
First I need to remove from the [**manifest**](https://github.com/diepquynh1501/local_manifests/blob/cm-13.0-grandprimeve3g/grandprimeve3g.xml#L25) ?

**Comment by ngoquang2708** [comment]
Try remove camera.sc8830 from your device/ tree.

**Comment by IKGapirov** [comment]
> Try remove camera.sc8830 from your device/ tree.

Thanks for answering.
I removed  [**this line**](https://github.com/diepquynh1501/android_device_samsung_grandprimeve3g/blob/cm-13.0/grandprimeve3g.mk#L56) and the compilation goes, is it not critical!?

**Comment by ngoquang2708** [comment]
That line for the camera. Without it, camera will not work.
-------- Original message --------From: Ikrom Gapirov <notifications@github.com> Date: 27/02/2019  13:46  (GMT+07:00) To: ngoquang2708/android_local_manifests <android_local_manifests@noreply.github.com> Cc: Quang Ngô <ngoquang2708@gmail.com>, Comment <comment@noreply.github.com> Subject: Re: [ngoquang2708/android_local_manifests] Build Lineage-13 (#1) 
Try remove camera.sc8830 from your device/ tree.

Thanks for answering.
I removed  this  @line and he compilation goes, is it not critical!?

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/ngoquang2708/android_local_manifests","title":"ngoquang2708/android_local_manifests","subtitle":"GitHub repository","main_image_url":"https://github.githubassets.com/images/email/message_cards/header.png","avatar_image_url":"https://github.githubassets.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/ngoquang2708/android_local_manifests"}},"updates":{"snippets":[{"icon":"PERSON","message":"@IKGapirov in #1: \r\n\r\n\r\n\r\n\u003e Try remove camera.sc8830 from your device/ tree.\r\n\r\nThanks for answering.\r\nI removed  [**this  @line**](https://github.com/diepquynh1501/android_device_samsung_grandprimeve3g/blob/cm-13.0/grandprimeve3g.mk#L56) and he compilation goes, is it not critical!? "}],"action":{"name":"View Issue","url":"https://github.com/ngoquang2708/android_local_manifests/issues/1#issuecomment-467744455"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/ngoquang2708/android_local_manifests/issues/1#issuecomment-467744455",
"url": "https://github.com/ngoquang2708/android_local_manifests/issues/1#issuecomment-467744455",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by IKGapirov** [comment]
Sorry but you said try remove camera.sc8830 from device/ tree..= (

**Comment by ngoquang2708** [comment]
Yeah. That just for the compile to pass successfully. Then you can figure out how to fix the camera.
-------- Original message --------From: Ikrom Gapirov <notifications@github.com> Date: 27/02/2019  13:52  (GMT+07:00) To: ngoquang2708/android_local_manifests <android_local_manifests@noreply.github.com> Cc: Quang Ngô <ngoquang2708@gmail.com>, Comment <comment@noreply.github.com> Subject: Re: [ngoquang2708/android_local_manifests] Build Lineage-13 (#1) Sorry but you said try remove camera.sc8830 from device/ tree..= (

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/ngoquang2708/android_local_manifests","title":"ngoquang2708/android_local_manifests","subtitle":"GitHub repository","main_image_url":"https://github.githubassets.com/images/email/message_cards/header.png","avatar_image_url":"https://github.githubassets.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/ngoquang2708/android_local_manifests"}},"updates":{"snippets":[{"icon":"PERSON","message":"@IKGapirov in #1: Sorry but you said try remove camera.sc8830 from device/ tree..= ("}],"action":{"name":"View Issue","url":"https://github.com/ngoquang2708/android_local_manifests/issues/1#issuecomment-467745825"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/ngoquang2708/android_local_manifests/issues/1#issuecomment-467745825",
"url": "https://github.com/ngoquang2708/android_local_manifests/issues/1#issuecomment-467745825",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
